tests/user_feedback.cpp

```diff
@@ -455,12 +455,12 @@ b = []
 	SECTION("tomlplusplus/issues/254") // https://github.com/marzer/tomlplusplus/issues/254
 	{
 		// Check constexpr support.
-		static_assert(toml::get_line(""sv, 1) == std::string_view{});
+		static_assert(!toml::get_line(""sv, 1));
```

Comment:
By the way, I originally wanted to use `std::nullopt`, both in the implementation (when out of range, `return std::nullopt`) and the unit test (above here, as in `static_assert(toml::get_line(""sv, 1) == std::nullopt)`, but I wasn't sure if it would be compatible with the `TOML_OPTIONAL_TYPE` option. What do you think?

Reply:
Yeah, that's correct. The library allows people to customize the `optional<T>` in use, so hard-coding `std::nullopt` would break that (e.g. some may be using `tl::optional` or similar).